pkgs/tools/typesetting/tex/blahtexml/default.nix

```diff
@@ -0,0 +1,42 @@
+{ fetchFromGitHub, lib, stdenv, libiconv, texlive, xercesc }:
+
+stdenv.mkDerivation {
+  pname = "blahtexml";
+  version = "0.9+date=2020-05-16";
+
+  src = fetchFromGitHub {
+    owner = "gvanas";
+    repo = "blahtexml";
+    rev = "92f2c5ff1f2b00a541b2222facc51ec72e5f6559";
+    hash = "sha256-ts+2gWsp7+rQu1US2/qEdbttB2Ps12efTSrcioZYsmE=";
+  };
+
+  outputs = [ "out" "doc" ];
+
+  nativeBuildInputs = [ texlive.combined.scheme-full ];
+  buildInputs = [ xercesc ] ++ lib.optionals stdenv.isDarwin [ libiconv ];
+
+  buildFlags =
+    [ "doc" ] ++
+    (if stdenv.isDarwin then
+      [ "blahtex-mac" "blahtexml-mac" ]
+    else [ "blahtex-linux" "blahtexml-linux" ]);
```

Comment:
```suggestion
    (if stdenv.isDarwin
     then [ "blahtex-mac" "blahtexml-mac" ]
     else [ "blahtex-linux" "blahtexml-linux" ]);
```

Reply:
Interesting, nixpkgs-fmt aligns `then` and `else` to `(` instead of `if`... a bug probably? Aligning `if/then/else` is definitely more readable.

Reply:
This is a bit strange. Here, Emacs aligned them in the same column...